app/scripts/views/global_header/search_box.js

```diff
@@ -0,0 +1,45 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define([
+  'backbone',
+  'views/base',
+  'stache!templates/global_header/search_box'
+], function (Backbone, BaseView, SearchBoxTemplate, router) {
+  'use strict';
+
+  var SearchBoxView = BaseView.extend({
+    template: SearchBoxTemplate,
+
+    events: {
+      'submit form': 'search'
+    },
+
+    search: function (event) {
+      event.preventDefault();
+
+      var encodedQuery = window.encodeURIComponent(this.$('.query').val());
+
+      Backbone.history.navigate('search/' + encodedQuery, { trigger: true });
```

Comment:
we probably want to eventually do something nice if the user empties out the search box and hits enter, but this is fine for now